lib/iris/_deprecation_helpers.py

```diff
@@ -24,10 +24,20 @@
 
 import warnings
 
+from iris.exceptions import IrisError
+
+
+class IrisDeprecation(IrisError, DeprecationWarning):
```

Comment:
Perhaps we should have two classes: one for [deprecations](https://docs.python.org/2/library/exceptions.html#exceptions.DeprecationWarning) and one for [changes in behaviour](https://docs.python.org/2/library/exceptions.html#exceptions.FutureWarning)?